### PR TITLE
Create cart-attachments-section-api-quick-start-guides.md

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [X] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [ ] Spelling and grammar accuracy (self-explanatory)

New .md created to list all Cart Attachments API Quick Start Guides.

### CODE REVIEW

File: docs/guides/Checkout/cart-attachments-section-api-quick-start-guides.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+This section contains the quick start guides of the endpoints described in the [Cart Attachments section](https://developers.vtex.com/docs/api-reference/checkout-api-overview#cart-attachments) of the Checkout API Reference.
```
Comment (karenkrieger) [inline docs/guides/Checkout/cart-attachments-section-api-quick-start-guides.md:1]: O link tá dando 404

**Review by karenkrieger — CHANGES_REQUESTED** [review]

File: docs/guides/Checkout/cart-attachments-section-api-quick-start-guides.md (hunk 1 above)
Comment (ricardoaerobr) [reply docs/guides/Checkout/cart-attachments-section-api-quick-start-guides.md]: Link atualizado para o novo arquivo. Deverá estar disponível após o builder.

Comment (ricardoaerobr) [reply docs/guides/Checkout/cart-attachments-section-api-quick-start-guides.md]: Link atualizado e já publicado: https://developers.vtex.com/docs/guides/checkout-api-overview#cart-attachments

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**